packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
 sidebar_position: 7
 ---
 
-# Travel Rule Implementations
+# Travel Rule Research
 
-The Verite open source examples include a demonstration of how exchanging verifiable credentials may be helpful in implementing a scalable mechanism for financial institutions to comply with Travel Rule regulations.
+The Verite open source repo includes some code that provides a cursory demonstration of how exchanging verifiable credentials could be a core component of scalable and open ecosystems  for financial institutions to comply with Travel Rule reporting obligations with minimal privacy, data leakage, and security risks.
 
-## About the Travel Rule
+In addition to this high-level "proof of concept," long-tail research in ongoing into FATF use cases forming a cornerstone of Verite in the latter part of 2023. To get involved with the research team, contact Verite's <a href=mailto:verite@centre.io>standards team at Centre</a>.  
```

Comment:
> In addition to this high-level "proof of concept," long-tail research in ongoing into FATF use cases forming a cornerstone of Verite in the latter part of 2023
Is this a complete sentence? maybe replace "long-tail research in" with "long-tail research is">

Reply:
correct interpretation, "in" was a typo for "is", although perhaps "research is ongoing" is both too passive-voice and too hard on non-native speakers for this context :D